modules/gdscript/gdscript_compiler.cpp

```diff
@@ -3040,6 +3042,132 @@ void GDScriptCompiler::make_scripts(GDScript *p_script, const GDScriptParser::Cl
 	}
 }
 
+GDScriptCompiler::FunctionLambdaInfo GDScriptCompiler::_get_function_replacement_info(GDScriptFunction *p_func, int p_index, int p_depth, GDScriptFunction *p_parent_func) {
+	FunctionLambdaInfo info;
+	info.function = p_func;
+	info.parent = p_parent_func;
+	info.script = p_parent_func;
+	info.name = p_func->get_name();
+	info.line = p_func->_initial_line;
+	info.index = p_index;
+	info.depth = p_depth;
+	//info.code_hash = hash_murmur3_buffer(reinterpret_cast<const int *>(p_func->code.ptr()), p_func->code.size() * sizeof(int));
+	//info.code_size = p_func->code.size();
+	info.capture_count = 0;
+	info.use_self = false;
+	info.arg_count = p_func->_argument_count;
+	info.default_arg_count = p_func->_default_arg_count;
+	//info.argument_types = p_func->argument_types;
+	//info.return_type = p_func->return_type;
```

Comment:
Any reason to keep these commented lines? If they're not needed they can just be omitted.